docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -30,6 +30,7 @@ The response looks like:
     "primary" : false
   },
   "assigned" : false,                             <1>
+  "shard_state_fetch_pending": false,
```

Comment:
Can you add a ` <1>` an an explanatory note below? (Something similar to what I wrote above why shard state/store fetching is relevant for allocation).